src/libstd/sys/windows/stdio.rs

```diff
@@ -116,8 +116,8 @@ impl Stdin {
 }
 
 impl Stdout {
-    pub fn new() -> Stdout {
-        Stdout(get(c::STD_OUTPUT_HANDLE).unwrap())
+    pub fn new() -> io::Result<Stdout> {
+        Stdout(try!(get(c::STD_OUTPUT_HANDLE)))
```

Comment:
I think this needs a surrounding `Ok` (above as well), perhaps jut a `get(...).map(|handle| ...)` ?